docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -253,3 +256,23 @@ Creates an exception that excludes all LFC-signed trusted processes:
 
 [role="screenshot"]
 image::images/nested-exp.png[]
+
+[float]
+[[manage-exceptions]]
+=== View and manage exceptions
+
+The Exception lists table enables you to view and manage all exceptions that have been assigned to rules. To view the Exception lists table, go to *Detections* -> *Manage detection rules*, then select the *Exception Lists* tab.
+
+[role="screenshot"]
+image::images/exception-list.png[]
+
+The table displays each Exception list on an individual row, with the most recently created list at the top. Each list contains information such as the number of rules the exception is assigned to, the name of the rule(s) assigned to the exception, the date created, the date last edited, and options to export or delete the list. Use the Search bar to enter a KQL query to filter the lists by `name`, `created_by`, or `list_id`.
+
+TIP: To view the details of the rule the exception is assigned to, click the link in the `Rules assigned to` column.
+
+To export or delete an Exception list, select the required action button on the appropriate list. 
```

Comment:
Should we add that if a list is linked to rules, they'll be removed and see the warning modal otherwise no modal is shown and list is simply deleted.